### PR TITLE
#159102576 Users should be able to signup or log in with social accounts

### PR DESCRIPTION
**Pull request by rovilay**
#### What does this PR do?
Allows users to signup or log in with their social accounts (google or facebook).
#### Description of Task to be completed?
* Add google login functionality
* Add facebook login functionality
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#159102576
#### Screenshots (if appropriate)
N/A
#### Questions
N/A

### CODE REVIEW

**Review by ozimos — APPROVED** [review]
Good work overall. But please leave formatting changes out of your PR next time (signin.html), or make a separate PR. this will make it easier to focus on your functional changes. Once again, good work. Keep it up

File: server/src/controllers/users.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { avatars } from './avatars';
+ import { avatars } from './avatars';
```
[the remark this answers is not in the document]
Comment (iverenshaguy) [reply server/src/controllers/users.js]: Could you please fix this?

[the remark this answers is not in the document]
Comment (rovilay) [reply server/src/controllers/users.js]: Ok... will do that now.

[the remark this answers is not in the document]
Comment (rovilay) [reply server/src/controllers/users.js]: done

**Pull request merged**